test/phoenix_test/playwright_test.exs

```diff
@@ -12,6 +12,31 @@ defmodule PhoenixTest.PlaywrightTest do
     end
   end
 
+  describe "screenshot/3" do
+    setup do
+      File.rm_rf("screenshots")
+      :ok
+    end
+
+    test "takes a screenshot of the current page as a PNG", %{conn: conn} do
```

Comment:
Probably coincidental: Other tests are now failing even in retry run with `--max-cases 1`